config/locales/en.yml

```diff
@@ -788,7 +789,7 @@ en:
         wac: "The WAC is responsible for maintaining feedback and communication between the WCA Community and the WCA Staff. They maintain the WCA Forums and summarize surveys and attempt to increase reach across the worldwide community."
     contacts:
       title: "WCA Contact Form"
-      faq_note_html: 'Before making an inquiry, you may wish to take a look at our <a href="/faq">Frequently Asked Questions</a>! If you wish to contact a specific committee or team not listed here, you can find their contact information on <a href="/teams-committees-councils">this page</a>.'
+      faq_note_html: 'Before making an inquiry, you may wish to take a look at our <a href="/faq">Frequently Asked Questions</a>! If you wish to contact a specific committee or team not listed here, you can find their contact information on <a href="/teams-committees-councils">this page</a> (Some committee/team might have also opted out from contacting, you will not be able to contact them).'
```

Comment:
This extra snippet of text should appear on the T/C overview page directly. Perhaps you can change the previously existing "you can find their contact information on" to something like "you can contact them on <a>this page</a>", which still holds true even with email opt-out (we don't guarantee that you will be able to contact them **via email**, but if they opt out you will still be able to ultimately contact them via the form)